index.html

```diff
@@ -0,0 +1,484 @@
+                  <a href="" class="icon-linkedin icon-social"></a>
+                  <a href="" class="icon-google-plus2 icon-social"></a>
+                  <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard.</p>
+                </figcaption>
+              </figure>
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h3 class="f-bold h3-lines">CLIENTS</h3>
+          <section class="l-client">
+            <figure class="l-client-item">
+              <img src="/img/matthew.png" alt="Client Matthew Broderic Photo" class="l-client-img">
+              <figcaption class="l-client-txt">
+                <p class="f-bold"><strong class="f-client">Matthew Broderic</strong> <span class="f-light">(CEO Hopicola Inc.)</span></p>
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard my text ever since the 1500s, when lorem upsum is the best text for you.</p>
+              </figcaption>
+            </figure>
+            <a href="" class="icon-circle-left s-icon-circle" role="button"></a>
+            <a href="" class="icon-circle-right s-icon-circle" role="button"></a>
+          </section>
+        </section>
+      </article>
+      <article class="b-orange" id="contact">
+        <h2 class="f-bold f-large">CONTACT</h2>
+        <section class="l-contact-txt">
+          <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+        </section>
+        <section class="l-flex-container">
+          <div class="l-flex-item l-flex-contact">
+            <form>
+              <input type="text" placeholder="Company name">
+              <input type="email" placeholder="Your@email.com">
+              <textarea class="s-form-message-field" placeholder="Message..."></textarea>
+              <br>
+              <button type="submit" class="btn s-form-btn s-form-btn-desktop">SEND MESSAGE</button>
+            </form>
+          </div>
+          <div class="l-flex-item l-flex-contact s-contact-info s-form-map-width">
+            <ul>
+              <li><span class="icon-mobile"></span> +970 879 580 (7:00 - 20:00)</li>
+              <li><span class="icon-mail"></span> <a href="mailto:info@company.com">info@company.com</a></li>
+              <li><span class="icon-pin"></span> <a href="">USA, San Francisco, 57010, Little Parry St. 826/31</a></li>
+            </ul>
+            <div class="s-arrow"></div>
+          </div>
+        </section>
+      </article>
+      <footer class="b-white l-footer">
+        <p>Theme by <a href="" class="s-theme-author">Pierre Borodin</a href=""> &copy; copyright 2012</p>
```

Comment:
Your ending tag has attributes in it (`</a href="">`) which is invalid and could cause problems.